angular/demo/src/app/samples/accordion/playground.route.ts

```diff
@@ -27,9 +28,9 @@ export default class PlaygroundComponent {
 
 	constructor() {
 		hashChangeHook((props) => {
-			this.widget?._widget.patch({...undefinedAccordionConfig, ...props});
+			this.widget?._widget.patch(getPropValues({...undefinedAccordionConfig, ...props}));
```

Comment:
Is calling `getPropValues` really useful here and line 33? Isn't it already called in `utils.ts`?

Reply:
you are right, I forgot to remove it